apps/src/netsim/NetSimTable.js

```diff
@@ -217,8 +210,7 @@ NetSimTable.prototype.subscribe = function () {
  * local channel. Also clears the saved callback.
  */
 NetSimTable.prototype.unsubscribe = function () {
-  this.channel_.unsubscribe(this.tableName_, this.channelCallback_);
-  this.channelCallback = undefined;
+  this.channel_.unsubscribe(this.tableName_);
```

Comment:
:/ We are technically unsubscribing the last callback associated with the `tableName_` key.  It's unsafe to subscribe multiple callbacks with the same key now - I guess that's not great, it just reflects the way we want to use it for now.